ui-library-filesharing-chat-composite/app/src/FileUploadHandler.tsx

```diff
@@ -0,0 +1,54 @@
+import { FileUploadHandler, FileUploadManager } from "@azure/communication-react";
+import axios from 'axios';
+import FormData from 'form-data';
+import { v4 } from 'uuid';
+
+const MAX_FILE_SIZE_MB = 50 * 1024 * 1024;
+const UNSUPPORTED_FILES = ['exe', 'bat', 'dat'];
+
+const fileUploadHandler: FileUploadHandler = (userId: string, fileUploads: FileUploadManager[], msToken?: string): void => {
+    for (const fileUpload of fileUploads) {
+      const fileExtension = fileUpload.file?.name.split('.').pop() || '';
+  
+      if (fileUpload.file && fileUpload.file?.size > MAX_FILE_SIZE_MB) {
+        fileUpload.notifyUploadFailed(`"${fileUpload.file?.name}" is too big. Select a file under 50MB.`);
+        continue;
+      }
+  
+      if (UNSUPPORTED_FILES.includes(fileExtension)) {
+        fileUpload.notifyUploadFailed(`Uploading ".${fileExtension}" files is not allowed.`);
+        continue;
+      }
+  
+      const uniqueFileName = `${v4()}-${fileUpload.file?.name}`;
+      const formData = new FormData();
+      formData.append('file', fileUpload.file, fileUpload.file?.name);
+      axios
+        .request({
+          method: 'post',
+          url: `http://localhost:7071/api/UploadFileToAzureBlobStore?filename=${uniqueFileName}`,
```

Comment:
Let's not hard code the development URL here.
You can just use `/api/UploadFileToAzureBlobStore?filename=${uniqueFileName}` and serve the entire project behind a proxy.

Please checkout the sample that I have merged with main
https://github.com/Azure-Samples/communication-services-javascript-quickstarts/tree/main/ui-library-filesharing-ui-components

It outlines the step needed to run both api and app behind a proxy that works without needing hardcoded URLs.